mautic/mautic
# Add stage name and stage date added to contacts report

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR added stage name and stage date_added to contacts reports.


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create contacts with various stages
3. Create report with columns of stages

![image](https://user-images.githubusercontent.com/462477/69904533-d7404100-13a7-11ea-8f61-ca89d3b61dc0.png)

4. Check if show properly
